sdk/runtime/runenv.go

```diff
@@ -171,8 +171,12 @@ func toBool(s string) bool {
 	return v
 }
 
+// toNet might parse any input, so it is possible to get an error and nil return value
 func toNet(s string) *IPNet {
-	_, ipnet, _ := net.ParseCIDR(s)
+	_, ipnet, err := net.ParseCIDR(s)
```

Comment:
Could maybe get rid of `toNet` and use `IPNet#UnmarshalJSON` instead, but that would not return a nil *IPNet?

Reply:
Let's revisit this after the release, as we might need a slightly bigger refactor of the env vars in these methods.